ocs_ci/utility/environment_check.py

```diff
@@ -112,35 +110,42 @@ def assign_get_values(
     env_status_dict[key] = items_filtered
 
 
-def get_environment_status(env_dict):
+def get_environment_status(env_dict, exclude_labels):
     """
     Get the environment status per kind in KINDS and save it in a dictionary
 
     Args:
         env_dict (dict): Dictionary that is a copy.deepcopy(ENV_STATUS_DICT)
+        exclude_labels (list): App labels to ignore leftovers
     """
     with ThreadPoolExecutor(max_workers=len(KINDS)) as executor:
         for key, kind in zip(env_dict.keys(), KINDS):
-            executor.submit(assign_get_values, env_dict, key, kind)
+            executor.submit(assign_get_values, env_dict, key, kind, exclude_labels=exclude_labels)
 
 
-def get_status_before_execution():
+def get_status_before_execution(exclude_labels):
     """
     Set the environment status and assign it into ENV_STATUS_PRE dictionary
+
+    Args:
+        exclude_labels (list): App labels to ignore leftovers
     """
-    get_environment_status(ENV_STATUS_PRE)
+    get_environment_status(ENV_STATUS_PRE, exclude_labels=exclude_labels)
 
 
-def get_status_after_execution():
+def get_status_after_execution(exclude_labels):
```

Comment:
I think `exclude_labels` should be set to `None` so existing usage will not break

Reply:
Thanks !.
The existing usage will not break because the function is called only from environment_checker where exclude_labels contain at least one value and it is passed to this function.
But I will set the default value to None because the function get_status_after_execution should be callable without exclude_labels param.

Reply:
Done

Reply:
nice, thanks